pkgs/development/python-modules/pki-core/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, buildPythonPackage, sphinx, rpmextract, requests,
+  six }:
+
+buildPythonPackage rec {
```

Comment:
Missing pythonImportsCheck and checkPhase

Reply:
Removed pki-core entirely, as it is not in use.